*** 2016-10-03 14:47:04.392 idempiere Log (CLogFile) ***
14:47:04.392 Adempiere.startup: iDempiere(r) Release 3.1_2015-10-31 -Smart Suite ERP, CRM and SCM- (c) 1999-2013 iDempiere(r); Implementation: iDempiere - Supported by iDempiere community [17]
14:47:04.396 Ini.loadProperties: E:\mercurialidempiere/idempiere.properties #33 [17]
14:47:04.983 AdempiereActivator.installPackage: ni.idempiere.importcurrencyrate 1.0.0 was installed: 2016-07-14 16:34:48.698 [17]
14:47:05.008 DB.isBuildOK: Build DB=${env.ADEMPIERE_VERSION} 20080428-1232 [17]
14:47:05.008 DB.isBuildOK: Build Cl=3.1.0.qualifier [17]
14:47:05.032-----------> DB.isBuildOK: Build Version Error 

The program assumes build version 3.1.0.qualifier, but database has build version ${env.ADEMPIERE_VERSION} 20080428-1232. 
This is likely to cause hard to fix errors. 
Please contact administrator. [17]
14:47:05.114 WebEnv.initWeb: Servlet Context Init Parameters: iDempiere Root
ServerRoot=1 [17]
14:47:05.132-----------> EMail.setFrom: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
14:47:05.134-----------> EMail.addTo: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
14:47:05.135-----------> EMail.isValid: From is invalid=null [17]
14:47:05.135-----------> EMail.isValid: From is invalid=null [17]
14:47:05.135-----------> MClient.sendEMail: Could NOT Send Email: Server started: iDempiere (odbz:172.16.9.71) to you @ company.org: Invalid Data (System) [17]
14:47:05.135 WebEnv.initWeb: Servlet Init Parameter: idempiereMonitor [17]
14:47:05.135 AdempiereMonitor.init:  [17]
14:47:06.708-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]
14:47:06.973-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://187.fwk1399961333:2/metainfo/zk/config.xml
with bundleresource://187.fwk1399961333:19/metainfo/zk/config.xml [17]
14:47:06.989-----------> ClassLocator.getDependentXMLResources: Replicate resource: zk
Overwrite bundleresource://187.fwk1399961333:11/metainfo/zk/config.xml
with bundleresource://187.fwk1399961333:28/metainfo/zk/config.xml [17]
14:47:06.993-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://187.fwk1399961333:12/metainfo/zk/config.xml
with bundleresource://187.fwk1399961333:29/metainfo/zk/config.xml [17]
14:47:07.024-----------> ClassLocator.getDependentXMLResources: Replicate resource: zul
Overwrite bundleresource://187.fwk1399961333:14/metainfo/zk/zk.xml
with bundleresource://187.fwk1399961333:31/metainfo/zk/zk.xml [17]
14:47:07.423-----------> LanguageDefinition.<init>: Replicated language: xhtml, overriden by [LanguageDefinition: xhtml] [17]
14:47:07.441-----------> LanguageDefinition.<init>: Replicated language: xul/html, overriden by [LanguageDefinition: xul/html] [17]
14:47:07.512-----------> ClassLocator.getDependentXMLResources: Replicate resource: calendar
Overwrite bundleresource://187.fwk1399961333:1/metainfo/zk/lang-addon.xml
with bundleresource://187.fwk1399961333:18/metainfo/zk/lang-addon.xml [17]
14:47:07.516-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://187.fwk1399961333:2/metainfo/zk/lang-addon.xml
with bundleresource://187.fwk1399961333:19/metainfo/zk/lang-addon.xml [17]
14:47:07.522-----------> ClassLocator.getDependentXMLResources: Replicate resource: gmapsz
Overwrite bundleresource://187.fwk1399961333:5/metainfo/zk/lang-addon.xml
with bundleresource://187.fwk1399961333:22/metainfo/zk/lang-addon.xml [17]
14:47:07.526-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeline
Overwrite bundleresource://187.fwk1399961333:6/metainfo/zk/lang-addon.xml
with bundleresource://187.fwk1399961333:23/metainfo/zk/lang-addon.xml [17]
14:47:07.526-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeplot
Overwrite bundleresource://187.fwk1399961333:7/metainfo/zk/lang-addon.xml
with bundleresource://187.fwk1399961333:24/metainfo/zk/lang-addon.xml [17]
14:47:07.542-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://187.fwk1399961333:12/metainfo/zk/lang-addon.xml
with bundleresource://187.fwk1399961333:29/metainfo/zk/lang-addon.xml [17]
14:47:07.557-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkplus
Overwrite bundleresource://187.fwk1399961333:13/metainfo/zk/lang-addon.xml
with bundleresource://187.fwk1399961333:30/metainfo/zk/lang-addon.xml [17]
14:47:07.717           WebUIServlet.init: iDempiere Web Client started successfully [17]
14:47:08.742-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]

*** 2016-10-03 14:47:08.974 idempiere Log (CLogFile) ***
14:47:10.334-----------> Node: [192.168.163.1]:5701 [adempiere.idempiere/localhost] [3.5.3] No join method is enabled! Starting standalone. [41]
15:37:26.219-----------> GridField.defaultFromSQLExpression: (SeqNo) SELECT COALESCE(MAX(Line),0)+10 AS DefaultValue FROM NI_NotificationParameter WHERE NI_NotificationMessage_ID=1000000 [33]
org.postgresql.util.PSQLException: ERROR: no existe la columna «line»
  Position: 21; State=42703; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:380)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:1418)
	at sun.reflect.GeneratedMethodAccessor21.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeQuery(Unknown Source)
	at org.compiere.model.GridField.defaultFromSQLExpression(GridField.java:745)
	at org.compiere.model.GridField.getDefaultValueByType(GridField.java:631)
	at org.compiere.model.GridField.getDefault(GridField.java:608)
	at org.compiere.model.GridField.getDefault(GridField.java:597)
	at org.compiere.model.GridField.getDefault(GridField.java:565)
	at org.compiere.model.GridTable.dataNew(GridTable.java:2616)
	at org.compiere.model.GridTab.dataNew(GridTab.java:1180)
	at org.adempiere.webui.adwindow.AbstractADWindowContent$20.onCallback(AbstractADWindowContent.java:1833)
	at org.adempiere.webui.adwindow.AbstractADWindowContent$20.onCallback(AbstractADWindowContent.java:1)

15:37:35.640-----------> GridField.defaultFromSQLExpression: (SeqNo) SELECT COALESCE(MAX(Line),0)+10 AS DefaultValue FROM NI_NotificationParameter WHERE NI_NotificationMessage_ID=1000000 [38]
org.postgresql.util.PSQLException: ERROR: no existe la columna «line»
  Position: 21; State=42703; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:380)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:1418)
	at sun.reflect.GeneratedMethodAccessor21.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeQuery(Unknown Source)
	at org.compiere.model.GridField.defaultFromSQLExpression(GridField.java:745)
	at org.compiere.model.GridField.getDefaultValueByType(GridField.java:631)
	at org.compiere.model.GridField.getDefault(GridField.java:608)
	at org.compiere.model.GridField.getDefault(GridField.java:597)
	at org.compiere.model.GridField.getDefault(GridField.java:565)
	at org.compiere.model.GridTable.dataNew(GridTable.java:2616)
	at org.compiere.model.GridTab.dataNew(GridTab.java:1180)
	at org.adempiere.webui.adwindow.AbstractADWindowContent$20.onCallback(AbstractADWindowContent.java:1833)
	at org.adempiere.webui.adwindow.AbstractADWindowContent$20.onCallback(AbstractADWindowContent.java:1)

16:35:03.331===========> DB.executeUpdate: INSERT INTO NI_Notification_Trigguer (AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,ni_notification_trigguer_ID,NI_NotificationMessage_ID,ni_notification_trigguer_UU) VALUES (?,?,?,?,?,?,?,?,?,?) [POSaveNI_Notification_T_8718b224-0c69-4c82-ad50-56393593a79d] [33]
org.postgresql.util.PSQLException: ERROR: el nuevo registro para la relación «ni_notification_trigguer» viola la restricción «check» «ninotificationtrigguer_tableprocess»
  Detail: La fila que falla contiene (1000000, 0, Y, 2016-10-03 16:35:03.307, 0, 2016-10-03, 0, null, null, null, 1000005, 1000000, 8f5d9a73-b0a4-4d3c-8f08-b309862a308b).; State=23514; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor52.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:1007)
	at org.compiere.model.PO.doInsert(PO.java:2974)
	at org.compiere.model.PO.saveNew(PO.java:2778)
	at org.compiere.model.PO.save(PO.java:2128)
	at org.compiere.model.GridTable.dataSavePO(GridTable.java:2223)
	at org.compiere.model.GridTable.dataSave(GridTable.java:1549)
	at org.compiere.model.GridTab.dataSave(GridTab.java:1007)

16:35:03.466===========> DB.saveError: DBExecuteError - ERROR: el nuevo registro para la relación «ni_notification_trigguer» viola la restricción «check» «ninotificationtrigguer_tableprocess»
  Detail: La fila que falla contiene (1000000, 0, Y, 2016-10-03 16:35:03.307, 0, 2016-10-03, 0, null, null, null, 1000005, 1000000, 8f5d9a73-b0a4-4d3c-8f08-b309862a308b). [33]
16:35:03.479-----------> GenericPO.doInsert: [POSaveNI_Notification_T_8718b224-0c69-4c82-ad50-56393593a79d]Not inserted - NI_Notification_Trigguer [33]
16:35:03.480===========> GenericPO.saveError: DBExecuteError - DBExecuteError:ERROR: el nuevo registro para la relación «ni_notification_trigguer» viola la restricción «check» «ninotificationtrigguer_tableprocess»
  Detail: La fila que falla contiene (1000000, 0, Y, 2016-10-03 16:35:03.307, 0, 2016-10-03, 0, null, null, null, 1000005, 1000000, 8f5d9a73-b0a4-4d3c-8f08-b309862a308b). [33]
16:35:03.481-----------> Msg.getMsg: NOT found: DBExecuteError:DBExecuteError:ERROR: el nuevo registro para la relación «ni_notification_trigguer» viola la restricción «check» «ninotificationtrigguer_tableprocess» [33]
16:35:03.481-----------> GridTable.saveWarning: DBExecuteError - DBExecuteError:DBExecuteError:ERROR: el nuevo registro para la relación «ni_notification_trigguer» viola la restricción «check» «ninotificationtrigguer_tableprocess»
  Detail: La fila que falla contiene (1000000, 0, Y, 2016-10-03 16:35:03.307, 0, 2016-10-03, 0, null, null, null, 1000005, 1000000, 8f5d9a73-b0a4-4d3c-8f08-b309862a308b). [33]
